tools/src/Packages.ts

```diff
@@ -63,13 +64,13 @@ export type UnimoduleJson = {
 export class Package {
   path: string;
   packageJson: PackageJson;
-  unimoduleJson: UnimoduleJson;
+  unimoduleJson: ExpoModuleConfig;
```

Comment:
```suggestion
  expoModuleConfig: ExpoModuleConfig;
```
I think we should rename that too.